docs/note_dialogs.rst

```diff
@@ -0,0 +1,60 @@
+A note about Modal Dialogs
+==========================
+
+Simple Dialogs
+--------------
+
+For QMessageBox.question one approach is to ``mock`` it:
+
+.. code-block:: python
+                
+    def test_Qt(qtbot, mock):
```

Comment:
This should be called `mocker`, `mock` is just a compatibility alias: https://github.com/pytest-dev/pytest-mock/issues/4

However, why require another plugin just for a doc example anyways? I think it'd be better to just use the built-in [monkeypatch](https://docs.pytest.org/en/latest/monkeypatch.html).